core/arch/arm/include/kernel/generic_boot.h

```diff
@@ -31,10 +31,12 @@
 
 #if defined(CFG_WITH_ARM_TRUSTED_FW)
 uint32_t cpu_on_handler(uint32_t a0, uint32_t a1);
-uint32_t *generic_boot_init_primary(uint32_t pageable_part);
+uint32_t *generic_boot_init_primary(uint32_t pageable_part, uint32_t unused,
+				    uint32_t fdt);
```

Comment:
Can't use paddr_t as it may potentially be a 64-bit type on a LPI32 system and this function is called from assembly. The best we can do is to change it to `unsigned long`,